ui/src/app/search-bar/search-bar.component.ts

```diff
@@ -48,5 +50,6 @@ export class SearchBarComponent implements OnInit {
   search(): void {
     // for now, only first tag entered is used for search
     this.dataService.fetchThreads(this.tags[0]);
+    this.router.navigate(['/threads']);
```

Comment:
(This comment is not in the scope of this PR)
dataService.fetchThreads() won't update until there is a successful response. Should we clear the thread list before anything returns from the backend?

Reply:
Yes, thread list should be cleared. Thanks for pointing out !